fixtures/with-side-effects/src/lib/atoms/atoms.ts

```diff
@@ -1,6 +1,6 @@
-import * as resetStyles from '../reset/reset.css';
 import type { RequiredResponsiveValue } from './sprinkles.css';
 import { sprinkles } from './sprinkles.css';
+import * as resetStyles from '../reset/reset.css';
```

Comment:
Should this stay above `sprinkles`, or does it not matter?

Reply:
This is caused by [eslint-plugin-import >= 2.27.5](https://github.com/import-js/eslint-plugin-import/blob/main/CHANGELOG.md#2275---2023-01-16)

Same issue reportei first noticed here https://github.com/seek-oss/sku/pull/735#discussion_r1102178164